### PR TITLE
DSD-1186: removing gap between content in TemplateAppContainer

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Fixes JIRA ticket [DSD-1186](https://jira.nypl.org/browse/DSD-1186)

## This PR does the following:

- Removes the gap between content inside the main element inside `TemplateAppContainer`.
- When there is no sidebar and no top or bottom content, the main content will span the full width. 

A few examples:
Removing row gap between top/bottom content and content spanning full width
Screen Shot 2023-07-03 at 4 21 56 PM

Content spanning full width when there is no sidebar
Screen Shot 2023-07-03 at 4 22 09 PM

Content spanning full width when there is no sidebar or top/bottom content
Screen Shot 2023-07-03 at 4 22 30 PM

One question, there are row gaps between `TemplateBreakout` which is mostly the header section, the main `TemplateContent`, and the `TemplateFooter`. Should those be removed?
Screen Shot 2023-07-03 at 4 16 36 PM

## How has this been tested?

Locally.

## Accessibility concerns or updates

- N/A

### Checklist:

- [ ] I have updated the Storybook documentation accordingly.
- [ ] I have added relevant accessibility documentation for this pull request.
- [ ] All new and existing tests passed.

### Front End Review:

- [ ] Review the Vercel preview deployment once it is ready.

### CODE REVIEW

**Comment by bigfishdesign13** [comment]
There was some misunderstanding.  The row gap should not have been removed from any of the elements.  This ticket was referring to what would be the column gap for the region that contains the `Mian Content` and `More Content` areas.  The issues is only seen when `sidebar` is set to `none` and `contentTop` or `contentBottom` have actual content.

**Comment by EdwinGuzman** [comment]
Makes sense. Updated: 
Screen Shot 2023-07-06 at 4 47 59 PM

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,10 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## Prerelease
 
+### Updates
+
+- Updates the spacing gap between main content in the `TemplateAppContainer` component.
```
Comment (bigfishdesign13) [inline CHANGELOG.md:19]: This should be rewitten as a `Fixes` item.

**Review by bigfishdesign13 — APPROVED** [review]
Looks good.  Just update the CHANGELOG entry and the `Latest Version`.

**Pull request merged**